### ISSUE TITLE
Train and Test dataset partition

### ISSUE THREAD

**Issue by hezt** [gifford-lab/DeepLigand#5]
Hello,

Could you please illustrate how to **partite the train set and the test set** from CV files (http://gerv.csail.mit.edu/deepligand_CVdata/) to get the **evaluation performance curve depicted in your paper**?

I'm trying to reimplement your train and evaluate processes.

Thanks,
Zitong

**Comment by hezt** [comment]
Hello,

Moreover, whether you concatenate the prediction results on each fold, where the model was trained on the other 4 folds, to draw auROC and auPRC curves?

Best,
Zitong

**Comment by haoyangz** [comment]
@hezt For each of the five folds, we trained one model using the other four folds before using it to predict on this fold. The resulting predictions of the five folds were concatenated to calculate auROC and other metrics.

**Comment by KiAkize** [comment]
Hello，

I am also trying to retrain 5cv models to reimplement results.

Could you please illustrate what each column of the downloaded 5CV data means?

In addition, besides renaming MHC names to the format in the MHC_pseudo.dat, what else needs to be done before using preprocess.py to transform training data?

Thank you!
